docs/analyzer/user_guide.md

```diff
@@ -793,9 +793,14 @@ optional arguments:
                         Set verbosity level.
 
 environment variables:
-  CC_ANALYZERS_FROM_PATH       Set to `yes` or `1` to enforce taking the
-                               analyzers from the `PATH` instead of the given
-                               binaries.
+  CC_ANALYZERS_FROM_PATH   Set to `yes` or `1` to enforce taking the analyzers
+                           from the `PATH` instead of the given binaries.
+  CC_CLANGSA_PLUGIN_DIR    If the CC_ANALYZERS_FROM_PATH environment variable
+                           is set you can configure the plugin directory of the
+                           Clang Static Analyzer by using this environment
+                           variable.
+  CC_SEVERITY_MAP_FILE     Path of the checker-severity mapping config file.
```

Comment:
Please extend the documentation of the `CodeChecker check` subcommand in this file too.